Readme.md

```diff
@@ -91,12 +93,39 @@ $ ./y.sh test --release
 
 ## Usage
 
+**Note:** You can ignore this step, by just cd into the repo directory.
+
 `$CG_GCCJIT_DIR` is the directory you cloned this repo into in the following instructions:
 
 ```bash
 export CG_GCCJIT_DIR=[the full path to rustc_codegen_gcc]
 ```
 
+Then you have to run these commands, in the corresponding order:
+
+```bash
+1- ./y.sh prepare
+2- ./y.sh build
+```
+If you get an error while running the second command, you should delete the first line inside of your ``config.toml`` like the following:
+
+```bash
+gcc-path = "gcc-build/gcc"
+download-gccjit = true
+```
+Delete ``gcc-path = "gcc-build/gcc"`` and run ``./y.sh build`` again.
+
+To check if all is  working correctly, run ``./y.sh cargo build --manifest-path tests/hello-world/Cargo.toml`` in your terminal.
+
+
+### Install rustup if need it
```

Comment:
I think this information should be at the top.

Reply:
But first we have to run the prepare and build commands as i understand. That is the reason that is not on top of that section.

Reply:
Euh... You need to rust to build any `y.sh` command.

Reply:
When you say at the top, which section do you mean?

Reply:
Before `Building`.

Reply:
> Euh... You need to rust to build any `y.sh` command.

Sorry, at the begining I understood something different.